test/DynamoCoreWpfTests/GraphLayoutTests.cs

```diff
@@ -361,6 +361,7 @@ public void GraphLayoutComplex()
         {
             OpenModel(GetDynPath("GraphLayoutComplex.dyn"));
             IEnumerable<NodeModel> nodes = ViewModel.CurrentSpace.Nodes;
+            nodes.ToList().ForEach(node => node.ReportPosition());
```

Comment:
@QilongTang @reddyashish - The reason for the test failure was basically that the pinned notes were in the wrong position when doing the auto layout, because notes are initialized with a size of 100x100 this doesn't get updated until the view renders the notes but as there is no view in this test i manually trigger the report position so the pinned notes are correct before we do the auto layout